pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_checking-xref-validation-errors.adoc

```diff
@@ -0,0 +1,21 @@
+[id="checking-xref-validation-errors_{context}"]
+
+= Checking xref validation errors
+
+[role="_abstract"]
+Check your content for xref validation errors. Invalid xrefs result in broken link when published. 
+
+.Prerequisites
+
+* The content has been imported to {ProductShortName}.
+* The supported forms of xref have been used.
++
+For information on the supported xref forms, see xref:about-xrefs-validation-in-pantheon_{context}[About xref validation in {ProductShortName}].
+
+.Procedure
+. Click the {ContentTerm} that has a red alert icon beside it.
```

Comment:
This isn't really a step in a procedure, in fact, this isn't really a procedure. If there's an alert icon, there's a validation error.
- Is there one icon per module/assembly if there are 1 or more xref errors? Or is there one icon per error?
- How do you know what the error is? What debug info does Pv2 tell you?

Reply:
@stoobie 
> Is there one icon per module/assembly if there are 1 or more xref errors? Or is there one icon per error?

On the search/home page where all the titles are listed, only a single icon is displayed for a title that contains one or more xref validation. However, when you click on a title and see the document details page, all the failing xrefs are listed under the validations section 

> How do you know what the error is? What debug info does Pv2 tell you?

On the document details page for the title that has xref validations a list of all the invalid xrefs is displayed. The list contains the target for each failing xref. Writers can search for this target in the adoc to debug which of the xrefs is failing